.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     FLAKE8=flake8
     CODECOV=codecov
     COVERAGE=coverage
+    PANDAS=pandas<0.21
```

Comment:
is there some problem with the latest version of pandas?

Reply:
I was getting ` Double requirement given: numpy==1.12.1 ... (already in numpy==1.9.3 )` and build was failing at [Travis](https://travis-ci.org/yt-project/yt/jobs/392083344#L652).
Solution: https://github.com/pandas-dev/pandas/issues/20723

Reply:
Ah, so it's because pandas dropped support for `python3.4` - can you only add this version restriction for the python3.4 builder?

To be honest, we should probably drop testing for python3.4, it's going EOL - I'll make a note to bring that up at the team meeting tomorrow.

Reply:
Our Travis lint stage is executed for python3.4, do you want me to update that too with python3.6?

Reply:
Yup.